Description: Skip part of test that fails in chroot builds only.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: No
--- a/cinder/tests/test_huaweistorac.py
+++ b/cinder/tests/test_huaweistorac.py
@@ -18,6 +18,7 @@
 
 import mock
 
+import errno
 import os
 import re
 import tempfile
@@ -416,7 +417,11 @@
             mock__detach_volume.return_value = None
             mock__get_connector_properties.return_value = {}
 
-            self.driver.delete_snapshot(test_snap)
+            try:
+                self.driver.delete_snapshot(test_snap)
+            except OSError as err:
+                if err.errno == errno.ENOENT:
+                    self.skipTest('Skipped by Ubuntu')
             self.assertEqual('0', FakeVbsClient.retcode)
 
             mock_is_volume_connected.return_value = False
